ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -190,6 +261,43 @@ pub(crate) struct Instruction {
     name: RefCell<Option<String>>,
 }
 
+impl PartialEq for Instruction {
+    fn eq(&self, other: &Self) -> bool {
+        std::ptr::eq(self, other)
+    }
+}
+
+impl Instruction {
+    pub(crate) fn get_operand(&self, index: usize) -> &Operand {
+        self.operands.get(index).unwrap()
```

Comment:
Isn't this `self.operands[index]` but more verbose? Also because of the implicit `panic` I think we should document the `# Panics` in the docstring.

Reply:
Do you mind if I also rename this to just `operand()` to be more rusty?

Reply:
Go ahead!

Reply:
Fixed in 9f91ce460fb9314f10f5f741bae136f5659b388f.

I forgot the rename, will push another commit shortly.

Reply:
Renamed in ac7f75e99b6094ce242655ec49c4bb51a3b00f8b.

Let's keep that commit separate, as it's partly existing code we are changing there.